python/main.py

```diff
@@ -1,51 +1,44 @@
-from flask import (Flask, request, jsonify, Response)
```

Comment:
I don't think this was being used. It may have been left there to make it easier to add additional features though.